core/modules/layout/layout.module

```diff
@@ -1812,6 +1812,35 @@ function layout_context_required_by_path($path) {
   return $required_contexts[$path];
 }
 
+/**
+ * Return the current user context, which is always available.
+ */
+function layout_current_user_context() {
+  global $user;
+
+  $current_user_context = layout_create_context('user', array(
+    'name' => 'current_user',
+    'label' => t('Current user'),
+    'locked' => TRUE,
+  ));
+
+  // Get a complete user entity
+  $context_info = layout_get_context_info('user');
+  if (isset($context_info['load callback']) && function_exists($context_info['load callback'])) {
+    // in case this has been set by other modules
```

Comment:
Coding standards (https://api.backdropcms.org/php-standards#comments) + move the comment outside the conditional:

```suggestion
  // In case this has been set by other modules.
  if (isset($context_info['load callback']) && function_exists($context_info['load callback'])) {
```

Reply:
Rewrote comment for both conditions.